pkg/config/config.go

```diff
@@ -125,6 +125,9 @@ const (
 	DeleteDelayTimeKey = "juicefs-delete-delay"
 	DeleteDelayAtKey   = "juicefs-delete-at"
 
+	// MountPointDevMinorKey mount pod annotation key to record the minor number from the fuse mountpoint.
+	MountPointDevMinorKey = "juicefs/mountpoint-dev-minor"
```

Comment:
It's only used by node driver, save it there should be enough.

Reply:
done